csodb.yml

```diff
@@ -0,0 +1,17 @@
+name: csodb
+channels:
+- conda-forge
+- digitalglobe
```

Comment:
Be careful when mixing channels. Unless you really need this channel for certain package. It's better to leave it out.